futurenhs.app/pages/_app.page.tsx

```diff
@@ -34,6 +34,29 @@ const CustomApp = ({ Component, pageProps }) => {
     let hasFormErrors: boolean = false
     let headTitle: string = pageProps.pageTitle || pageProps.contentText?.title
 
+    useEffect(() => {
+
+        if (layoutId !== layoutIds.GROUP) {
```

Comment:
Would prefer we avoided this conditional statement - it may build into something more cumbersome and inflexible as the app grows. Can we set the focus instead declaratively within the tabs component?

Reply:
Ah apologies, just spotted this suggestion on the bug/ticket. Yes shouldn't be a problem, it would still very briefly focus on the body initially, would that be an issue?

Reply:
Removed condition, body is now always focused. Setting focus to the tabs is now handled in the group page header & tabbed nav component